lib/storage/data/file/backend.js

```diff
@@ -0,0 +1,113 @@
+import { errors, stringHash } from 'arsenal';
```

Comment:
seems this file should be moved over in a separate PR in which you create a dataFile server